Fix warning in bget.c

I see this one warning when I build:

/home/ed/pio3/ParallelIO/src/clib/bget.c: In function ‘bget’:
/home/ed/pio3/ParallelIO/src/clib/bget.c:778:74: warning: format ‘%x’ expects argument of type ‘unsigned int’, but argument 6 has type ‘void *’ [-Wformat=]
                 printf("%s %d memory request exceeds block size %ld %ld %x\n",__FILE__,__LINE__,
                                                                          ^

--- a/src/clib/bget.c
+++ b/src/clib/bget.c
@@ -775,8 +775,8 @@ bufsize requested_size;
                 buf =  (void *) (bdh + 1);
 
                 /*only let this happen once */
-                printf("%s %d memory request exceeds block size %ld %ld %x\n",__FILE__,__LINE__,
-                       size,exp_incr,buf);
+                printf("%s %d memory request exceeds block size %ld %ld\n",__FILE__,__LINE__,
+                       size,exp_incr);
                 exp_incr = size+sizeof(struct bhead);
 
                 return buf;

